lib/decompress.ps1

```diff
@@ -180,7 +180,12 @@ function Expand-InnoArchive {
         $Removal
     )
     $LogPath = "$(Split-Path $Path)\innounp.log"
-    $ArgList = @('-x', "-d`"$DestinationPath`"", "-c`{app`}\$ExtractDir", "`"$Path`"", '-y')
+    $ArgList = @('-x', "-d`"$DestinationPath`"", "`"$Path`"", '-y')
```

Comment:
I think single quotes could be used instead of escaping double quote.

```suggestion
    $ArgList = @('-x', "-d'$DestinationPath'", "'$Path'", '-y')
```

Reply:
I'm not sure if single quote will expand the variables, please wait for my testing.

Reply:
It is inside double quote.

Powershell should evaluate the variable first and then pass the string with evaluated value as argument. So it should be called as "innounp -x .. -d'actualPath\whatever'"

Reply:
I've tested single quote, and it will break the extraction... Escaped double quotes are MUST here...

And I also notice that, for `-cDIR`, if `DIR` has white spaces, it needs double quotes too, but commonly there won't be any white spaces for either `{app}\xxx` or `{xxx}`. If meet so, I'll make a hotfix.